test/quantization/core/test_config.py

```diff
@@ -0,0 +1,323 @@
+# Owner(s): ["oncall: quantization"]
```

Comment:
nit: test_config.py -> test_backend_config.py?

Reply:
Ok. I originally wanted this test file for the other configs as well but maybe we should put them in separate test files